Configuration/Sets/SitePackage/config.yaml

```diff
@@ -0,0 +1,5 @@
+name: t3docs/site-package
+label: Example site package set
```

Comment:
Don't know if the label has to be "Site Package" since you called the directory in Sets "SitePackage".

Reply:
Only the label will be displayed in the select field in the sitepackages module